windows.md

```diff
@@ -0,0 +1,36 @@
+---
+title: zipline-live
+description: On premise live trading with zipline
+---
+# zipline-live installaion on Windows
+
+This tutorial assumes you have a clean Windows installation with Administrator rights. This tutorial has been implemented using Windows 10 on [Microsoft Azure Cloud](https://azure.microsoft.com).
+ 
+## Install Anaconda
+Download and install [Anaconda](https://www.anaconda.com/download/) (Python 2.7).
+
+## Install Microsoft Visual C++ Compiler for Python 2.7
+Download and install from https://www.microsoft.com/en-us/download/details.aspx?id=44266
```

Comment:
The link is not clickable, I'd suggest using:
`[here](https://www.microsoft.com/en-us/download/details.aspx?id=44266)`